recipe/meta.yaml

```
@@ -22,9 +22,9 @@ requirements:
     - wheel
   run:
     - python
-    - black >=22.3.0
+    - black >=23.11.0
     - python-lsp-server >=1.4.0
-    - toml
+    - toml  # [py<311]
```

```suggestion
    - tomli  # [py<311]
```
https://github.com/python-lsp/python-lsp-black/blob/v2.0.0/setup.cfg#L24

---

Ooh, good catch! Thanks